### PR TITLE
Add Translation strings and updated po files

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

**Comment by ostcar** [comment]
ok

File: openslides/motions/static/js/motions/base.js [hunk 1]
```diff
@@ -312,5 +312,33 @@ angular.module('OpenSlidesApp.motions', ['OpenSlidesApp.users'])
...
+])
+
+
+// Mark all motion workflow state strings for translation in JavaScript.
+// (see motions/signals.py)
+.config([
+    'gettext',
+    function (gettext) {
+        // workflow 1
+        gettext('Simple Workflow'),
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/base.js]: Do we need the comma at the end of the line? See also all the lines below.

File: openslides/motions/static/js/motions/site.js [hunk 2]
```diff
@@ -979,6 +979,6 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
             );
         };
     }
-]);
+])
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: I think this semicolon was correct.

**Comment by normanjaeckel** [comment]
Two questions. Else looks good.

File: openslides/motions/static/js/motions/base.js (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/motions/static/js/motions/base.js]: comma not required. Removed.

File: openslides/motions/static/js/motions/site.js (hunk 2 above)
Comment (emanuelschuetze) [inline openslides/motions/static/js/motions/site.js]: Fixed.

**Comment by emanuelschuetze** [comment]
All done. Rebased. Ready for merge.

**Pull request merged**